pycir/cir.py

```diff
@@ -0,0 +1,97 @@
+# -*- coding: utf-8 -*-
+"""
+Centered isotonic regression, generic but more tailored to dose-response and dose-finding data
+
+Assaf Oron, recoded from R 'cir' package
+"""
```

Comment:
Renamed from "code" to "pycir" since the parent folder name is what the module will be importable as!